src/components/UpdateProfile/UserPassword.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import styles from './updateProfile.module.scss';
+
+class UserPassword extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentPassword: '',
+      newPassword: '',
+      rNewPassword: '',
+    };
+  }
+
+  onChange = event => {
+    this.setState({ [event.target.name]: event.target.value });
+  }
+
+  onSubmit = e => {
+    e.preventDefault();
```

Comment:
Did you send the update password request here?

Reply:
No. I initially was supposed to work on the update password before the plan changed. At that moment, I had already started implementing the feature but I had to stop thereafter.

Reply:
Ok. That makes sense.